### PR TITLE
[WIP] Json11 cpp library

### PR DESCRIPTION
**Pull request by cdalvaro**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source json11`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict json11` (after doing `brew install json11`)?

-----

### CODE REVIEW

File: Formula/json11.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class Json11 < Formula
+  desc "Tiny JSON library for C++11"
+  homepage "https://github.com/dropbox/json11"
+  url "https://github.com/dropbox/json11.git", :revision => "ec4e45219af1d7cde3d58b49ed762376fccf1ace"
+  version "1.0.0"
```
Comment (SMillerDev) [inline Formula/json11.rb]: Is there an officially tagged release?

**Comment by fxcoudert** [comment]
Hi @cdalvaro, thanks for the pull request. For us to consider including json11 as Homebrew formula, we need it to have stable, released versions with clear versioning scheme: https://docs.brew.sh/Acceptable-Formulae We are, for now, going to decline this pull request.

**Comment by cdalvaro** [comment]
Thank you for your comment @fxcoudert!

This library is built and maintained by Dropbox and their last commit was on Jun 21, 2017.
It is a really stable library.

The version is annotated in the CMakeLists.txt file. (`project(json11 VERSION 1.0.0 LANGUAGES CXX)`) although they don't have any release in their [repository on GitHub](https://github.com/dropbox/json11/releases) neither tags.

Could be possible to accept this pull request despite the inconveniences?

File: Formula/json11.rb (hunk 1 above)
Comment (cdalvaro) [reply Formula/json11.rb]: No... 😞

**Comment by SMillerDev** [comment]
From the acceptable formula requirements:
> We don’t accept software without a tagged version because they regularly break due to upstream changes and we can’t provide bottles for them.

It really just breaks a part of homebrew if we would do that.
You could set it up in a personal tap though.

**Comment by cdalvaro** [comment]
I'll try to get a tag or release version of the library.

Thank you for your attention @SMillerDev!

**Comment by cdalvaro** [comment]
Finally there is a [1.0.0 version](https://github.com/dropbox/json11/releases) for the library.

Please @SMillerDev, could you reopen this PR in order to update the Homebrew formula?

Thank you very much!

**Comment by SMillerDev** [comment]
I think it'd be better to make a new pull request.

**Pull request closed without merging**